config/initializers/mas_cms_client.rb

```diff
@@ -0,0 +1,9 @@
+Mas::Cms::Client.config do |c|
+  c.timeout =  ENV['HTTP_REQUEST_TIMEOUT'].to_i
+  c.open_timeout =  ENV['HTTP_REQUEST_TIMEOUT'].to_i
+  c.api_token =  ENV['MAS_CMS_API_TOKEN']
+  c.host =  ENV['MAS_CMS_URL']
```

Comment:
Should this env be other for Fincap CMS like `ENV['FINCAP_CMS_URL']`?

Reply:
I think it can remain as `ENV['MAS_CMS_URL']` as they will be using the same CMS application but different databases.

Reply:
The url will be different from MAS CMS, so it should be Fincap CMS.